lua/core/mods.lua

```diff
@@ -16,7 +16,7 @@ function Mods.scan(root, pattern)
   local p = pattern or Mods.search_pattern
 
   local matches, error = norns.system_glob(r .. p)
```

Comment:
If the underlying call is changed to throw an error then I think it is reasonable for this function to can catch the error, print out the error, and return `{}` as a fallback value.